CONTRIBUTING.md

```diff
@@ -67,7 +67,7 @@ for source changes.
 
 Prerequisites:
 
-* go >= 1.19
```

Comment:
Actually think this could just match the version defined in `go.mod`, instead of keeping up with latest.

Reply:
Problem this creates is of course the `-compat=$ver` which we do in the `Makefile`.